kotti_content_proxy/__init__.py

```diff
@@ -17,3 +17,11 @@ def includeme(config):
 def kotti_configure(settings):
     settings['pyramid.includes'] += ' kotti_content_proxy'
     settings['kotti.available_types'] += ' kotti_content_proxy.resources.ContentProxy'  # noqa
+
+    proxied_attrs = settings.get('kotti_content_proxy.proxied_attributes', '')
+    proxied_attrs = set(proxied_attrs.split())
+
+    from .resources import ContentProxy
```

Comment:
Having this at module level led to circular import in the tests (starting with imports in the `conftest.py`).